### ISSUE TITLE
Post Functionality

### ISSUE THREAD

**Issue by maektwain** [muellners/ltcim#5]

**Comment by tacitonic** [comment]
Working on blog post functionality

**Comment by theupscale** [comment]
Wonderful

**Comment by maektwain** [comment]
Hey , How are you doing ? with the task.

**Comment by tacitonic** [comment]
I'm having issues assigning parameters like title, content, etc. to the respective values in the stack theme. I have pushed the layouts that are to be used @maektwain

**Comment by maektwain** [comment]
I think you need to define , the post layout did you do it ?

**Comment by maektwain** [comment]
You have to create the post layout in the layout section and then put the theme there and then use 

```
{% page.title %}

```

**Comment by tacitonic** [comment]
Post layout is defined. There are other values than `page.title` as well

**Comment by theupscale** [comment]
What kind of values you are talking about.?

**Comment by tacitonic** [comment]
I could implement a basic post layout and I have pushed it as of now.
But I'll need time for integrating a complex layout.

**Comment by theupscale** [comment]
What do you mean by complex layout?

**Comment by tacitonic** [comment]
This is the template in stack that I'm thinking can be used for blog post page. It lists the posts categorically.

![variant-blog-list-simple](https://user-images.githubusercontent.com/38654946/82459097-22284f00-9ad5-11ea-8e5a-935f32f29a61.jpg)

And this is the template for the post itself.

![variant-blog-article-simple](https://user-images.githubusercontent.com/38654946/82459246-513ec080-9ad5-11ea-8ee4-dbb54b4829d2.jpg)

So, as this template has multiple components, I'm unable to integrate Jekyll and these components. Could you help with this @maektwain ?

**Comment by maektwain** [comment]
Which components are you referring to? I think Jekyll support few times start with?

**Comment by maektwain** [comment]
I can confirm Jekyll support 

Title, tags, images and content 😊

Let me know if you are facing an issue with the code convention?

**Comment by maektwain** [comment]
```
---
layout: default
---
<div class="post">

  <header class="post-header">
    <h1 class="post-title">{{ page.title }}</h1>
  </header>

  <article class="post-content">
    {{ content }}
  </article>

</div>
```

**Comment by tacitonic** [comment]
> Let me know if you are facing an issue with the code convention?
Yes, I'm having difficulties in the template code convention specifically. I have commented out the html in `_layouts/post.html` and `/blog.html`. It is causing more problems when I implement by this method.
